### PR TITLE
add ppdet model zoo document file

### PR DESCRIPTION
**Pull request by Noplz**

### CODE REVIEW

File: PaddleCV/object_detection/doc/MODEL_ZOO.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+- NCCL 2.1.2
+
+## Common settings
+
+- All models below except SSD were trained on `coco_2017_train`, and tested on the `coco_2017_val`.
+- The Batch Normalization layer of the backbone is replaced by the Affine Channel layer.
+- Unless otherwise noted, all ResNet are adopted [ResNet-B](https://arxiv.org/pdf/1812.01187) as the backbone.
+- For RCNN and RetinaNet models, only horizontal flipping data augmentation was used in the training phase and no augmentations were used in the testing phase.
+
+##Training Schedules
```
Comment (jerrywgz) [inline PaddleCV/object_detection/doc/MODEL_ZOO.md]: ##后面加空格

File: PaddleCV/object_detection/doc/MODEL_ZOO.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+- We adopt the exactly same training schedules as [Detectron](https://github.com/facebookresearch/Detectron/blob/master/MODEL_ZOO.md#training-schedules). 
+- 1x indicates the schedule starts at a LR of 0.02 and is decreased by a factor of * 0.1 after 60k and 80k iterations and finally terminates at 90k iterations for minibatch size 16. 
+- 2x indicates twice as long as the 1x schedule with the LR change points scaled proportionally.
+
+##ImageNet Pretrained Models
```
Comment (jerrywgz) [inline PaddleCV/object_detection/doc/MODEL_ZOO.md]: 同上

File: PaddleCV/object_detection/doc/MODEL_ZOO.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+- All models below except SSD were trained on `coco_2017_train`, and tested on the `coco_2017_val`.
+- The Batch Normalization layer of the backbone is replaced by the Affine Channel layer.
+- Unless otherwise noted, all ResNet are adopted [ResNet-B](https://arxiv.org/pdf/1812.01187) as the backbone.
+- For RCNN and RetinaNet models, only horizontal flipping data augmentation was used in the training phase and no augmentations were used in the testing phase.
+
+##Training Schedules
+
+- We adopt the exactly same training schedules as [Detectron](https://github.com/facebookresearch/Detectron/blob/master/MODEL_ZOO.md#training-schedules). 
+- 1x indicates the schedule starts at a LR of 0.02 and is decreased by a factor of * 0.1 after 60k and 80k iterations and finally terminates at 90k iterations for minibatch size 16.
```
Comment (jerrywgz) [inline PaddleCV/object_detection/doc/MODEL_ZOO.md]: 我们有些模型1x对应的是18w轮，lr=0.01

Comment (Noplz) [reply PaddleCV/object_detection/doc/MODEL_ZOO.md]: 嗯 这个最后有写，对应的是batchsize 16的情况，detectron中也是这样描述的。

File: PaddleCV/object_detection/doc/MODEL_ZOO.md (hunk 1 above)
Comment (Noplz) [reply PaddleCV/object_detection/doc/MODEL_ZOO.md]: Done

File: PaddleCV/object_detection/doc/MODEL_ZOO.md (hunk 2 above)
Comment (Noplz) [reply PaddleCV/object_detection/doc/MODEL_ZOO.md]: Done

File: PaddleCV/object_detection/doc/MODEL_ZOO.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
+# Model Zoo and Benchmark
```
Comment (qingqing01) [inline PaddleCV/object_detection/doc/MODEL_ZOO.md]: doc -> docs

**Review by qingqing01 — APPROVED** [review]

File: PaddleCV/object_detection/doc/MODEL_ZOO.md (hunk 4 above)
Comment (Noplz) [reply PaddleCV/object_detection/doc/MODEL_ZOO.md]: done

**Review by heavengate — APPROVED** [review]
LGTM, I'll add model zoo of yolov3 after this PR merged

File: PaddleCV/object_detection/docs/MODEL_ZOO.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+- Python 2.7.1
+- PaddlePaddle 1.5
+- CUDA 9.0
+- CUDNN 7.4
+- NCCL 2.1.2
+
+## Common settings
+
+- All models below except SSD were trained on `coco_2017_train`, and tested on the `coco_2017_val`.
```
Comment (willthefrog) [inline PaddleCV/object_detection/docs/MODEL_ZOO.md]: the `coco_2017_val` -> `coco_2017_val`

File: PaddleCV/object_detection/docs/MODEL_ZOO.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+- The Batch Normalization layer of the backbone is replaced by the Affine Channel layer.
```
Comment (willthefrog) [inline PaddleCV/object_detection/docs/MODEL_ZOO.md]: Batch Normalization layers in backbones are replaced by Affine Channel layers

File: PaddleCV/object_detection/docs/MODEL_ZOO.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+- Unless otherwise noted, all ResNet are adopted [ResNet-B](https://arxiv.org/pdf/1812.01187) as the backbone.
```
Comment (willthefrog) [inline PaddleCV/object_detection/docs/MODEL_ZOO.md]: all ResNet backbones adopt the [ResNet-B](https://arxiv.org/pdf/1812.01187) variant.

File: PaddleCV/object_detection/docs/MODEL_ZOO.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+- For RCNN and RetinaNet models, only horizontal flipping data augmentation was used in the training phase and no augmentations were used in the testing phase.
+
+## Training Schedules
+
+- We adopt the exactly same training schedules as [Detectron](https://github.com/facebookresearch/Detectron/blob/master/MODEL_ZOO.md#training-schedules). 
+- 1x indicates the schedule starts at a LR of 0.02 and is decreased by a factor of * 0.1 after 60k and 80k iterations and finally terminates at 90k iterations for minibatch size 16. 
+- 2x indicates twice as long as the 1x schedule with the LR change points scaled proportionally.
```
Comment (willthefrog) [inline PaddleCV/object_detection/docs/MODEL_ZOO.md]: 2x schedule is twice as long as 1x,  with the LR milestones scaled accordingly.

File: PaddleCV/object_detection/docs/MODEL_ZOO.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+
+## ImageNet Pretrained Models
+
+The backbone models pretrained on ImageNet are available. All models are trained on the standard ImageNet-1k dataset. All pretrained models can be download [here](https://github.com/PaddlePaddle/models/tree/develop/PaddleCV/image_classification#supported-models-and-performances)
```
Comment (willthefrog) [inline PaddleCV/object_detection/docs/MODEL_ZOO.md]: All backbone models are pretrained on standard ImageNet-1k dataset and can be downloaded

File: PaddleCV/object_detection/docs/MODEL_ZOO.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+## Common settings
+
+- All models below except SSD were trained on `coco_2017_train`, and tested on the `coco_2017_val`.
+- The Batch Normalization layer of the backbone is replaced by the Affine Channel layer.
+- Unless otherwise noted, all ResNet are adopted [ResNet-B](https://arxiv.org/pdf/1812.01187) as the backbone.
+- For RCNN and RetinaNet models, only horizontal flipping data augmentation was used in the training phase and no augmentations were used in the testing phase.
+
+## Training Schedules
+
+- We adopt the exactly same training schedules as [Detectron](https://github.com/facebookresearch/Detectron/blob/master/MODEL_ZOO.md#training-schedules).
```
Comment (willthefrog) [inline PaddleCV/object_detection/docs/MODEL_ZOO.md]: the exactly -> exactly the

File: PaddleCV/object_detection/docs/MODEL_ZOO.md [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+
+- All models below except SSD were trained on `coco_2017_train`, and tested on the `coco_2017_val`.
+- The Batch Normalization layer of the backbone is replaced by the Affine Channel layer.
+- Unless otherwise noted, all ResNet are adopted [ResNet-B](https://arxiv.org/pdf/1812.01187) as the backbone.
+- For RCNN and RetinaNet models, only horizontal flipping data augmentation was used in the training phase and no augmentations were used in the testing phase.
+
+## Training Schedules
+
+- We adopt the exactly same training schedules as [Detectron](https://github.com/facebookresearch/Detectron/blob/master/MODEL_ZOO.md#training-schedules). 
+- 1x indicates the schedule starts at a LR of 0.02 and is decreased by a factor of * 0.1 after 60k and 80k iterations and finally terminates at 90k iterations for minibatch size 16.
```
Comment (willthefrog) [inline PaddleCV/object_detection/docs/MODEL_ZOO.md]: \* 0.1 -> 10

Comment (willthefrog) [reply PaddleCV/object_detection/docs/MODEL_ZOO.md]: finally -> eventually

File: PaddleCV/object_detection/docs/MODEL_ZOO.md [hunk 12]
```diff
@@ -0,0 +1,81 @@
...
+
+The backbone models pretrained on ImageNet are available. All models are trained on the standard ImageNet-1k dataset. All pretrained models can be download [here](https://github.com/PaddlePaddle/models/tree/develop/PaddleCV/image_classification#supported-models-and-performances)
+
+## Baselines
+
+### Faster & Mask R-CNN
+
+| Backbone             | Type           | Img/gpu | Lr schd | Box AP | Mask AP |                           Download                           |
+| :------------------- | :------------- | :-----: | :-----: | :----: | :-----: | :----------------------------------------------------------: |
+| ResNet50             | Faster         |    1    |   1x    |  35.1  |    -    | [model](https://paddlemodels.bj.bcebos.com/object_detection%2Ffaster_rcnn_r50_1x.tar) |
```
Comment (jerrywgz) [inline PaddleCV/object_detection/docs/MODEL_ZOO.md]: All urls should replace '%2F' by '/'

File: PaddleCV/object_detection/docs/MODEL_ZOO.md [hunk 13]
```diff
@@ -0,0 +1,81 @@
+# Model Zoo and Benchmark
+## Environment
+
+- Python 2.7.1
+- PaddlePaddle 1.5
+- CUDA 9.0
+- CUDNN 7.4
```
Comment (qingqing01) [inline PaddleCV/object_detection/docs/MODEL_ZOO.md:7]: cuDNN >= 5.0

File: PaddleCV/object_detection/docs/MODEL_ZOO.md [hunk 14]
```diff
@@ -0,0 +1,81 @@
+# Model Zoo and Benchmark
+## Environment
+
+- Python 2.7.1
```
Comment (qingqing01) [inline PaddleCV/object_detection/docs/MODEL_ZOO.md:4]: Python >= 2.7.1 ?

File: PaddleCV/object_detection/docs/MODEL_ZOO.md (hunk 11 above)
Comment (qingqing01) [inline PaddleCV/object_detection/docs/MODEL_ZOO.md]: 1x 这里要声明下 bs=8时，是从0.01开始吗？

File: PaddleCV/object_detection/docs/MODEL_ZOO.md (hunk 9 above)
Comment (qingqing01) [reply PaddleCV/object_detection/docs/MODEL_ZOO.md]: Need to add explanation for ResNet50_cos_pretrained for models based on R50.

File: PaddleCV/object_detection/docs/MODEL_ZOO.md (hunk 12 above)
Comment (willthefrog) [reply PaddleCV/object_detection/docs/MODEL_ZOO.md]: yes! I was tripped by this too!!

File: PaddleCV/object_detection/docs/MODEL_ZOO.md (hunk 6 above)
Comment (Noplz) [reply PaddleCV/object_detection/docs/MODEL_ZOO.md]: Done

File: PaddleCV/object_detection/docs/MODEL_ZOO.md (hunk 13 above)
Comment (Noplz) [reply PaddleCV/object_detection/docs/MODEL_ZOO.md:7]: 这里列举的我理解应该是测试环境，不是运行的要求

File: PaddleCV/object_detection/docs/MODEL_ZOO.md (hunk 14 above)
Comment (Noplz) [reply PaddleCV/object_detection/docs/MODEL_ZOO.md:4]: 同上

File: PaddleCV/object_detection/docs/MODEL_ZOO.md (hunk 7 above)
Comment (Noplz) [reply PaddleCV/object_detection/docs/MODEL_ZOO.md]: done

File: PaddleCV/object_detection/docs/MODEL_ZOO.md (hunk 8 above)
Comment (Noplz) [reply PaddleCV/object_detection/docs/MODEL_ZOO.md]: done

File: PaddleCV/object_detection/docs/MODEL_ZOO.md (hunk 11 above)
Comment (Noplz) [reply PaddleCV/object_detection/docs/MODEL_ZOO.md]: done

File: PaddleCV/object_detection/docs/MODEL_ZOO.md (hunk 12 above)
Comment (Noplz) [reply PaddleCV/object_detection/docs/MODEL_ZOO.md]: Done

File: PaddleCV/object_detection/docs/MODEL_ZOO.md (hunk 10 above)
Comment (Noplz) [reply PaddleCV/object_detection/docs/MODEL_ZOO.md]: done

File: PaddleCV/object_detection/docs/MODEL_ZOO.md (hunk 5 above)
Comment (Noplz) [reply PaddleCV/object_detection/docs/MODEL_ZOO.md]: done

File: PaddleCV/object_detection/docs/MODEL_ZOO.md [hunk 15]
```diff
@@ -9,39 +9,41 @@
...
 - For RCNN and RetinaNet models, only horizontal flipping data augmentation was used in the training phase and no augmentations were used in the testing phase.
 
 ## Training Schedules
 
-- We adopt the exactly same training schedules as [Detectron](https://github.com/facebookresearch/Detectron/blob/master/MODEL_ZOO.md#training-schedules). 
-- 1x indicates the schedule starts at a LR of 0.02 and is decreased by a factor of * 0.1 after 60k and 80k iterations and finally terminates at 90k iterations for minibatch size 16. 
-- 2x indicates twice as long as the 1x schedule with the LR change points scaled proportionally.
+- We adopt exactly the same training schedules as [Detectron](https://github.com/facebookresearch/Detectron/blob/master/MODEL_ZOO.md#training-schedules). 
+- 1x indicates the schedule starts at a LR of 0.02 and is decreased by a factor of 10 after 60k and 80k iterations and eventually terminates at 90k iterations for minibatch size 16 or starts at a LR of 0.01 and is decreased by a factor of 10 after 120k and 160k iterations and eventually terminates at 180k iterations for minibatch size 8.
```
Comment (willthefrog) [inline PaddleCV/object_detection/docs/MODEL_ZOO.md]: man...this is lazy...

File: PaddleCV/object_detection/docs/MODEL_ZOO.md (hunk 11 above)
Comment (Noplz) [reply PaddleCV/object_detection/docs/MODEL_ZOO.md]: Done

File: PaddleCV/object_detection/docs/MODEL_ZOO.md (hunk 15 above)
Comment (willthefrog) [reply PaddleCV/object_detection/docs/MODEL_ZOO.md]: For batch size 8, LR is decreased to 0.01, total training iterations are doubled, and the decay milestones are scaled by 2.

**Review by willthefrog — APPROVED** [review]
LGTM
